zebra-chain/src/block/tests/generate.rs

```diff
@@ -182,8 +170,8 @@ fn single_transaction_block_many_inputs(oversized: bool) -> Block {
     // A block header
     let (block_header, block_header_bytes) = block_header();
 
-    // A LockTime
-    let (lock_time, lock_time_bytes) = lock_time();
+    let lock_time = LockTime::Time(DateTime::from_timestamp(61, 0).unwrap());
```

Comment:
Optional/Nitpick: Do we want to move this to a constant?

Reply:
I don't think so. It's an arbitrary value used only in test code, without much significance.